itests/src/test/resources/testconfiguration.properties

```diff
@@ -156,6 +156,7 @@ minillap.query.files=\
   select_transform_hint.q,\
   skip_header_footer_aggr.q,\
   skip_header_footer_proj.q,\
+  smblimit.q,\
```

Comment:
Not sure why we need to run this explicitly for minillap? Its going to be tested against miniLocallLlap anyway right?

Reply:
because this test easily reproduced the failure I fixed, only on minillap (so on hdfs, not local files)
I'm about to create a separate qtest, emphasizing that it's about to prove this solution

Reply:
Thanks! Makes sense now!